README.md

```diff
@@ -0,0 +1,198 @@
+Classy Test
+===========
+
+Meteor package which provides a class-based wrapper around `tinytest`.
+
+It has the following features:
+
+ * Class-based test cases.
+ * Common setUp/tearDown methods for test cases, separated for server and client side.
+ * A single test can interleave client-side and server-side assertions.
+ * Support for asynchronous tests.
+ * Compatible with tinytest (all tests are actually registered via tinytest).
+
+Installation
+------------
+
+```
+meteor add peerlibrary:classy-test
+```
+
+Test cases
+----------
+
+Each test case is a class extending from `ClassyTestCase` as follows:
+
+```coffeescript
+class SimpleTestCase extends ClassyTestCase
+  testThatTrueIsTrue: =>
+    @assertTrue true, "True should be true."
+
+  testThatFalseIsFalse: =>
+    @assertFalse false, "False should be false."
+
+# Register the test case.
+ClassyTestCase.addTest new SimpleTestCase()
+```
+
+This simple test case definition will generate two tests via tinytest, the first will be called `SimpleTestCase - ThatTrueIsTrue` and the second one `SimpleTestCase - ThatFalseIsFalse`.
```

Comment:
Or should it be `@name`? I am not sure if you can use `name` though. Because it is reserved word, even if it is not used in Firefox. Like, in conflicts things. :-)